tests/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ list(TRANSFORM SOURCES PREPEND ${CMAKE_CURRENT_SOURCE_DIR}/)
 foreach(SRC ${SOURCES})
   # add_vsyncer_check works as a normal add_test, but the command runs inside
```

Comment:
Now the comment in the first line seems outdated

Reply:
done